### PR TITLE
End date and time added to forms and displayed

### LINKED ISSUE #432: Creating an end time and multi-day meetups
## Description
As an organizer,
I need to create meetups,
so that I can organize multi-day hackathons or day long workshops.

## Mocks
N/A

### Update [Required]
- [ ] Meetups need a start time and an end time.
- [ ] Meetups need a start day and end day.

### Enhancement to Update [Optional]
- [ ] Multiple times for different days.

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

## Estimation
20 hours

### PR DESCRIPTION
**Pull request by Tlazypanda**
### Description
End date and time attributes kept optional and added to the form.After creation of meetup these attributes are displayed in meetup.html.

## MOCKS

Before-
screenshot from 2018-11-14 01-00-08
image

After-
screenshot from 2018-11-09 18-55-27
screenshot from 2018-11-09 18-55-50

Fixes #432 #286 

### Type of Change:

- Code

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### How Has This Been Tested?
Tested locally screenshots attached..

### Checklist:
**Delete irrelevant options.**

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [X] My changes generate no new warnings

### CODE REVIEW

File: systers_portal/templates/meetup/meetup.html [hunk 1]
```diff
@@ -3,10 +3,16 @@
 {% block meetup_location_page_content %}
   <h2 id="meetup-title">{{ meetup.title }}</h2>
   <div class="box-container box-body pt25">
-    <p>
```
Comment (abhi20161997) [inline systers_portal/templates/meetup/meetup.html]: changes like these shouldn't be reflected.

File: systers_portal/templates/meetup/meetup.html [hunk 2]
```diff
@@ -3,10 +3,16 @@
...
   <h2 id="meetup-title">{{ meetup.title }}</h2>
   <div class="box-container box-body pt25">
-    <p> 
+    <p>
       <b> Date: </b> {{ meetup.date }}
-      &nbsp; &nbsp; 
+      {% if meetup.end_date %}
+      <b> - </b> {{ meetup.end_date }}
+      {% endif %}
+      &nbsp; &nbsp;
```
Comment (abhi20161997) [inline systers_portal/templates/meetup/meetup.html]: have you checked whether the responsiveness is getting affected by this or not ?

File: systers_portal/templates/meetup/meetup.html [hunk 3]
```diff
@@ -27,7 +33,7 @@ <h2 id="meetup-title">{{ meetup.title }}</h2>
           <td>{{ not_coming_no }}</td>
         </tr>
       </tbody>
-    </table> 
+    </table>
```
Comment (abhi20161997) [inline systers_portal/templates/meetup/meetup.html]: again this shouldn't be reflected.

**Comment by abhi20161997** [comment]
@Tlazypanda  Also mocks should be included like before scenario and after scenario- include mocks for how the forms looked before as well.

File: systers_portal/templates/meetup/meetup.html (hunk 2 above)
Comment (Tlazypanda) [reply systers_portal/templates/meetup/meetup.html]: @abhi20161997 Responsiveness is not affected works fine for all devices.

**Comment by sammy1997** [comment]
@Tlazypanda Please resolve the merge conflicts and update your codebase. This is a stale PR. Also, if possible add some tests!! The coverage seems to have decreased

**Comment by sammy1997** [comment]
@Tlazypanda Please let me know if you will be working on this. Or else, I can close the PR and make the issue available for others

**Comment by sammy1997** [comment]
@abhi20161997 @sakshi1499 should we close this? I had tagged the contributor but didn't get a response

**Comment by sakshi1499** [comment]
@sammy1997 I think we should.

**Comment by abhi20161997** [comment]
@sammy1997 , Yes ! It should be closed.

**Pull request closed without merging**